run-postinsts: add ptest to ptest-packagelists

dylan-matthew-turner opened the pull request:
This is a script that we have and it produces a warning because run-postints-ptest is not a part of the larger ptest-packagelists.inc file. We need to add it to a variable in that file.

Warning:

```
WARNING: run-postinsts-1.0-r10 do_package_qa: QA Issue: supports ptests but is not included in oe-core's ptest-packagelists.inc [missing-ptest]
```

It should be noted we are appending the variable that is in that file instead of adding run-postints-ptest directly as run-postints is a part of our changes, not openembedded-core

amstewart commented on recipes-devtools/run-postinsts/run-postinsts_%.bbappend:
```diff
@@ -29,3 +29,5 @@ do_install_ptest:append() {
 
 
 RDEPENDS:${PN}-ptest += "bash"
+PTESTS_FAST += "${PN}-ptest"
```
Could you move this line up to line 12 - per [the styleguide](https://github.com/ni/meta-nilrt/blob/nilrt/master/kirkstone/docs/styleguide.md#layout)?

dylan-matthew-turner committed:
run-postinsts: add ptest to ptest-packagelists

This is a script that we have and it produces a warning because
run-postints-ptest is not a part of the larger ptest-packagelists.inc
file. We need to add it to a variable in that file.

It should be noted we are appending the variable that is in that file
instead of adding run-postints-ptest directly as run-postints is a part
of our changes, not openembedded-core

Signed-off-by: Dylan Turner <dylan.turner@ni.com>
